### ISSUE TITLE
Artifact: A Symbol of Unity is missing

### ISSUE THREAD

**Issue by ulvric** [Apen/epicseven#39]
Noticed this artifact was missing even though newer artifacts are present such as Spear of a New Dawn.

**Comment by Apen** [comment]
Yes, i'm depedant of an API who provide all the datas, and it is not up to date at the moment
